### PR TITLE
Makes README cleaner

### PR DESCRIPTION
**Pull request by akshaybhalotia**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -7,6 +8,10 @@ To know more about Razorpay payment flow and steps involved, please read up here
 
 ## Installation
 
+This has 3 steps: add to project, installation and linking iOS SDK.
```
Comment (selvagsz) [inline README.md]: I'd recommend the following headers
- Installation
- Linking
   - automatic linking 
   - manual linking
   - Linking IOS sdk

Comment (akshaybhalotia) [inline README.md]: Linking at so many places seems confusing to me. That is why I tried to play with the words a little. The last step is a very important one and is certainly linking in the true sense. Could you suggest some other terminology that fits here?

Comment (selvagsz) [inline README.md]: `Linking` is the std term. http://facebook.github.io/react-native/releases/0.36/docs/linking-libraries-ios.html

May be, we can have "Linking IOS sdk" to "Extra steps for linking iOS sdk"

Comment (captn3m0) [inline README.md]: Been some time here. Can we close this now?

Comment (akshaybhalotia) [inline README.md]: @selvagsz Check now?

**Pull request merged**